vispy/visuals/volume.py

```diff
@@ -728,6 +756,8 @@ def method(self, method):
         # Get rid of specific variables - they may become invalid
         if 'u_threshold' in self.shared_program:
             self.shared_program['u_threshold'] = None
+        elif 'u_attenuation' in self.shared_program:
```

Comment:
Can this maybe be made an `if` instead of `elif` for future flexibility? Otherwise this assumes that two shaders will never use both parameters. I know, nit-pick, but still.